Allow to re-scramble duplicate cubes at MBLD (#887)
thewca/wca-regulations: wca-guidelines.md

## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving

> - H1+) [ADDITION] If a duplicate cube is found and during an attempt, the cube may be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.

Reviewer: Multi-Blind only exists for 3x3x3 right now, but we usually use "puzzle" when possible.
```suggestion
- H1+) [ADDITION] If a duplicate puzzle is found during an attempt, the cube puzzle be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.
```
